### PR TITLE
Code cleaning - removed redundant variables, unused imports, etc.

### PR DESCRIPTION
**Pull request by mcimbora**

### CODE REVIEW

File: core/src/main/java/org/radargun/config/ConfigSchemaGenerator.java [hunk 1]
```diff
@@ -63,7 +63,6 @@ protected void generate() {
       addAttribute(masterComplex, ATTR_PORT, intType, null, false);
 
       Element clustersChoice = createChoice(benchmarkSequence, 0, 1);
-      Element localComplex = createComplexElement(clustersChoice, ELEMENT_LOCAL, 0, 1);
```
Comment (rvansa) [inline core/src/main/java/org/radargun/config/ConfigSchemaGenerator.java]: createComplexElement has a side-effect. You probably want just to remove the unused variable. The same for scenarioType.

File: core/src/main/java/org/radargun/config/Stage.java [hunk 2]
```diff
@@ -15,7 +15,7 @@
    static final String NO_DEPRECATED_NAME = "__no_deprecated_name__";
    String name() default CLASS_NAME_WITHOUT_STAGE;
    String deprecatedName() default NO_DEPRECATED_NAME;
-   String doc();
+   String doc() default "";
```
Comment (rvansa) [inline core/src/main/java/org/radargun/config/Stage.java]: Not having a default value is intended, in order to force you to write some documention.

File: core/src/main/java/org/radargun/stages/AbstractDistStage.java [hunk 3]
```diff
@@ -26,7 +26,7 @@
  *
  * @author Mircea Markus &lt;Mircea.Markus@jboss.com&gt;
  */
-@Stage(doc = "")
```
Comment (rvansa) [inline core/src/main/java/org/radargun/stages/AbstractDistStage.java]: The documentation for this stage won't be ever seen in schema, therefore it's empty.

File: core/src/main/java/org/radargun/stages/cache/IsolationLevelCheckStage.java [hunk 4]
```diff
@@ -81,6 +81,7 @@ public DistStageAck executeOnSlave() {
       try {
          Thread.sleep(duration);
       } catch (InterruptedException e) {
+         log.error("Thread has been interrupted", e);
```
Comment (rvansa) [inline core/src/main/java/org/radargun/stages/cache/IsolationLevelCheckStage.java]: You should probably restore interrupt status as well.

File: plugins/infinispan51/src/main/java/org/radargun/service/Infinispan51Operations.java [hunk 5]
```diff
@@ -74,7 +74,7 @@ public boolean remove(K key) {
       @Override
       public V getAndRemove(K key) {
          boolean startedTx = startTxAndLock(key);
-         boolean retval = super.remove(key);
```
Comment (rvansa) [inline plugins/infinispan51/src/main/java/org/radargun/service/Infinispan51Operations.java]: Seems like you found a bug - you should not call both super.remove and super.getAndRemove. Could you fix that? Also, I think that stopTx should be called in finally block.

**Comment by rvansa** [comment]
And this will need a rebase, too.

**Comment by mcimbora** [comment]
@rvansa updated

**Pull request merged**